crates/celestia-grpc-client/src/proto/celestia.zkism.v1.rs

```diff
@@ -0,0 +1,286 @@
+    #[prost(bytes = "vec", tag = "8")]
+    pub state_membership_vkey: ::prost::alloc::vec::Vec<u8>,
+}
+/// MsgCreateZKExecutionISMResponse is the response type for CreateZKExecutionISM.
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct MsgCreateZkExecutionIsmResponse {
+    #[prost(string, tag = "1")]
+    pub id: ::prost::alloc::string::String,
+}
+/// MsgUpdateZKExecutionISM is the request type for UpdateZKExecutionISM.
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct MsgUpdateZkExecutionIsm {
+    /// ism identifier
+    #[prost(string, tag = "1")]
+    pub id: ::prost::alloc::string::String,
+    /// height is the Celestia height associated with the state transition update.
+    #[prost(uint64, tag = "2")]
+    pub height: u64,
+    /// proof is the ZK proof bytes (groth16).
+    #[prost(bytes = "vec", tag = "3")]
+    pub proof: ::prost::alloc::vec::Vec<u8>,
+    /// the public values used for proof verification.
+    #[prost(bytes = "vec", tag = "4")]
+    pub public_values: ::prost::alloc::vec::Vec<u8>,
+    /// the tx signer address
+    #[prost(string, tag = "5")]
+    pub signer: ::prost::alloc::string::String,
+}
+/// MsgUpdateZKExecutionISMResponse is the response type for UpdateZKExecutionISM.
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct MsgUpdateZkExecutionIsmResponse {
+    /// updated ism trusted state root (hex-encoded)
+    #[prost(string, tag = "1")]
+    pub state_root: ::prost::alloc::string::String,
+    /// updated ism trusted height
+    #[prost(uint64, tag = "2")]
+    pub height: u64,
+}
+/// MsgSubmitMessages is the request type for SubmitMessages.
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct MsgSubmitMessages {
+    /// ism identifier
+    #[prost(string, tag = "1")]
+    pub id: ::prost::alloc::string::String,
+    /// height is the EVM application height associated with the state transition update.
+    #[prost(uint64, tag = "2")]
+    pub height: u64,
+    /// proof is the ZK proof bytes (groth16).
+    #[prost(bytes = "vec", tag = "3")]
+    pub proof: ::prost::alloc::vec::Vec<u8>,
+    /// the public values used for proof verification.
+    #[prost(bytes = "vec", tag = "4")]
+    pub public_values: ::prost::alloc::vec::Vec<u8>,
+    /// the tx signer address
+    #[prost(string, tag = "5")]
+    pub signer: ::prost::alloc::string::String,
+}
+/// MsgSubmitMessagesResponse is the response type for SubmitMessages.
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, Copy, PartialEq, ::prost::Message)]
+pub struct MsgSubmitMessagesResponse {}
+/// MsgProcessMessage is the request type for ProcessMessage.
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct MsgProcessMessage {
```

Comment:
This file is code-generated, we shouldn't be manually adding things here.

I have resolved conflicts and will push a change including this into the new build system